### PR TITLE
1102 fix form texts and placeholders

### LINKED ISSUE #1102: [TASK] Fix form texts and placeholders on all pages
### Description
All form texts, labels and placeholders need to be reviewed and make necessary changes.

Instead of "Upload an icon" put "Upload icon"

### Acceptance Criteria
All texts in all pages are clear.

### PR DESCRIPTION
**Pull request by minump**
## Description
Updated contribute page form field labels and placeholders.

Let me know if anything needs to be changed.
For now only contributions/create page form labels are changed.

**Fixes #1102**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: contributions/catalog/webapps/templates/contribute/contribute.html [hunk 1]
```diff
@@ -142,12 +142,12 @@ <h2 id="contribute">Contribution Details</h2>
                         {% if is_editable %}
                             <input class="form-control" id="contribution_Admins"
                                    name="contribution_contributionAdmins"
-                                   placeholder="Give a list of admin names, separated by comma"
+                                   placeholder="Give a list of github usernames, separated by comma, to be selected as admins"
```
Comment (sandeep-ps) [inline contributions/catalog/webapps/templates/contribute/contribute.html:145]:
```suggestion
                                   placeholder="Give a list of GitHub usernames, separated by a comma, to be selected as admins of this Contribution"
```

File: contributions/catalog/webapps/templates/contribute/contribute.html [hunk 2]
```diff
@@ -1247,10 +1247,8 @@ <h3 id="talent">Add a Talent to this Contribution</h3>
                             <label class="control-label col-sm-2">Data Description</label>
                             <div class="col-sm-10">
                                 <textarea id="data_des" name="talent_dataDescription"
-                                       placeholder="Detailed description of the data that is collected by this Talent, how it is used, and how this relates to the privacy level."
-                                       style="height:150px"
-                                       type="text">
-                                </textarea>
+                                       placeholder="Detailed description of the data that is collected by this Talent, how it is used, and how this relates to the privacy level"
```
Comment (sandeep-ps) [inline contributions/catalog/webapps/templates/contribute/contribute.html:1250]:
```suggestion
                                       placeholder="Provide a detailed description of the data collected by this Talent, how it is used, and how it relates to the Rokwire privacy levels."
```

File: contributions/catalog/webapps/templates/contribute/contribute.html
Comment (sandeep-ps) [inline contributions/catalog/webapps/templates/contribute/contribute.html:1]: Please change the placeholder texts for the following entries: 

Database Detail -> "Provide the name, version, and other details of the database server used."
Version -> "Latest Capability version number."
Health Check URL -> "Add a health check URL to check the Capability status continuously."

**Review by sandeep-ps — COMMENTED** [review]
I have left a few comments. The rest looks good for now. Thanks.

File: contributions/catalog/webapps/templates/contribute/contribute.html
Comment (minump) [reply contributions/catalog/webapps/templates/contribute/contribute.html:1]: updated

File: contributions/catalog/webapps/templates/contribute/contribute.html (hunk 2 above)
Comment (minump) [reply contributions/catalog/webapps/templates/contribute/contribute.html]: updated

File: contributions/catalog/webapps/templates/contribute/contribute.html [hunk 3]
```diff
@@ -210,34 +210,34 @@ <h3 id="capability">Add a Person to this Contribution</h3>
                             </div>
                         </div>
                         <div class="form-group">
-                            <label class="control-label col-sm-2">Contributor's first name</label>
+                            <label class="control-label col-sm-2">First Name</label>
                             <div class="col-sm-10">
-                                <input class="form-control" id="contributor_fname" name="person_firstName" placeholder="" type="text">
+                                <input class="form-control" id="contributor_fname" name="person_firstName" placeholder="Contributor's first name" type="text">
```
[the remark this answers is not in the document]
Comment (minump) [reply contributions/catalog/webapps/templates/contribute/contribute.html]: updated

File: contributions/catalog/webapps/templates/contribute/contribute.html [hunk 4]
```diff
@@ -142,12 +142,12 @@ <h2 id="contribute">Contribution Details</h2>
...
                             <input class="form-control" id="contribution_Admins"
                                    name="contribution_contributionAdmins"
-                                   placeholder="Give a list of admin names, separated by comma"
+                                   placeholder="Give a list of github usernames, separated by comma, to be selected as admins"
                                    type="text" value="{{ post.contributionAdmins|join(', ') }}">
                         {% else %}
                             <input class="form-control" id="contribution_Admins"
                                    name="contribution_contributionAdmins"
-                                   placeholder="Give a list of admin names, separated by comma"
+                                   placeholder="Give a list of github usernames, separated by comma, to be selected as admins"
```
[the remark this answers is not in the document]
Comment (minump) [reply contributions/catalog/webapps/templates/contribute/contribute.html]: updated

File: contributions/catalog/webapps/templates/contribute/contribute.html (hunk 1 above)
Comment (minump) [reply contributions/catalog/webapps/templates/contribute/contribute.html]: updated

**Comment by minump** [comment]
Placeholder texts updated according to feedback.
Ready for review. @sandeep-ps

File: contributions/catalog/webapps/templates/contribute/contribute.html [hunk 5]
```diff
@@ -210,34 +210,34 @@ <h3 id="capability">Add a Person to this Contribution</h3>
...
                         </div>
-                    </div>
-                    <div class="form-group">
-                        <label class="control-label col-sm-2">Contributor Email</label>
-                        <div class="col-sm-10">
-                            <input class="form-control" name="person_email" placeholder="" type="email">
+                        <div class="form-group">
+                            <label class="control-label col-sm-2">Email</label>
+                            <div class="col-sm-10">
+                                <input class="form-control" name="person_email" placeholder="Contributor's email" type="email">
```
Comment (sandeep-ps) [inline contributions/catalog/webapps/templates/contribute/contribute.html:233]:
```suggestion
                                <input class="form-control" name="person_email" placeholder="Person's email address" type="email">
```

File: contributions/catalog/webapps/templates/contribute/contribute.html
Comment (sandeep-ps) [reply contributions/catalog/webapps/templates/contribute/contribute.html:1]: Thanks.

File: contributions/catalog/webapps/templates/contribute/contribute.html [hunk 6]
```diff
@@ -210,34 +210,34 @@ <h3 id="capability">Add a Person to this Contribution</h3>
...
                         </div>
-                    </div>
-                    <div class="form-group">
-                        <label class="control-label col-sm-2">Contributor Phone</label>
-                        <div class="col-sm-10">
-                            <input class="form-control" name="person_phone" placeholder="" type="tel" minlength="9" maxlength="14">
+                        <div class="form-group">
+                            <label class="control-label col-sm-2">Phone Number</label>
+                            <div class="col-sm-10">
+                                <input class="form-control" name="person_phone" placeholder="Contributor's phone number (9 to 14 digits)" type="tel" minlength="9" maxlength="14">
```
Comment (sandeep-ps) [inline contributions/catalog/webapps/templates/contribute/contribute.html:239]:
```suggestion
                                <input class="form-control" name="person_phone" placeholder="Person's phone number (9 to 14 digits)" type="tel" minlength="9" maxlength="14">
```

**Review by sandeep-ps — APPROVED** [review]
We can merge this set of placeholder text changes. I have started editing a few more in Google Docs, but it still needs to be prepared for review and implementation.

**Comment by minump** [comment]
Merging to develop

**Pull request merged**